scripts/globals/combat/physical_utilities.lua

```diff
@@ -932,7 +934,7 @@ xi.combat.physical.handleBlock = function(defender, attacker, damage)
         end
     end
 
-    return damage
+    return damage, didBlock
```

Comment:
I'm not a big fan of creating functions that return 2 parameters

Then again, I'm also not a big fan of functions that return a converted value instead of a factor to convert the value.

I would think that, if the value fed is different than the value returned, the block did happen though

Reply:
^ That's a good point. Is it always the case that a block will result in less damage? Because you can use the value changing as your "did a block happen" signal. It does seem like it's cleaner to just have a function that works out "did I block"? and then apply any changes that need to happen because of the block after - rather than combining them and having this double return

Reply:
Thanks for the suggestions! The updated PR now refactors the `handleBlock` function and splits it into two functions `isBlocked` and `getDamageReductionForBlock` so that it is more consistent with other functions (like `isParried`) and removes the need for two return parameters. The damage reduction value is a flat reduction that is then subtracted from the hit damage.